projects/pgai/pgai/cli.py

```diff
@@ -104,6 +104,13 @@ def get_vectorizer(db_url: str, vectorizer_id: int) -> Vectorizer:
             api_key_name = embedding["api_key_name"]
             api_key = os.getenv(api_key_name, None)
             if api_key is None:
+                cur.execute(
```

Comment:
Could you add some (debug?) logging so that we can see where the vectorizer got the secret from? Something like this diff:

```diff
diff --git a/projects/pgai/pgai/cli.py b/projects/pgai/pgai/cli.py
index c890dff..fedccbb 100644
--- a/projects/pgai/pgai/cli.py
+++ b/projects/pgai/pgai/cli.py
@@ -103,13 +103,17 @@ def get_vectorizer(db_url: str, vectorizer_id: int) -> Vectorizer:
         if "api_key_name" in embedding:
             api_key_name = embedding["api_key_name"]
             api_key = os.getenv(api_key_name, None)
-            if api_key is None:
+            if api_key is not None:
+                log.debug(f"obtained secret '{api_key_name}' from environment")
+            else:
                 cur.execute(
                     "select ai.reveal_secret(%s)",
                     (api_key_name,),
                 )
                 row = cur.fetchone()
                 api_key = row["reveal_secret"] if row is not None else None
+                if api_key is not None:
+                    log.debug(f"obtained secret '{api_key_name}' from database")
             if not api_key:
                 raise ApiKeyNotFoundError(
                     f"api_key_name={api_key_name} vectorizer_id={vectorizer_id}"
```